content/pages/home/header.json

```diff
@@ -1,16 +1,14 @@
 {
   "body": "Find or publish datasets and AI algorithms for solving challenges in mobility. Experience and leverage decentralized technologies and the European Gaia-X data sharing concepts. Facilitating public-private, cross-sector, cross-border collaboration while maintaining digital sovereignty.",
   "cta": [
+    { "label": "Learn more", "link": "#" },
```

Comment:
this link is not working

Reply:
At the moment there is no design/page or decision to where they should go

Reply:
So what should be the next step for this? having buttons that don't do anything without a next action to take is not very ideal

Reply:
Maria will have a meeting with Maarten today and possibly with Niels to define it. If they don't come up with a solution we can always create a ticket to fix it. This ticket was messy from the beginning.

Reply:
"Learn more" link was removed from the design. So I removed it from the code as well. I think it's fixed now.